packages/core/src/scenes/Scene.ts

```diff
@@ -301,4 +302,9 @@ export interface Scene<T = unknown> {
    * Should always return `true`.
    */
   isCached(): boolean;
+
+  /**
+   * Should this scene be rendered below the previous scene during a transition?
+   */
+  belowPrevious: SimpleSignal<boolean, this>;
```

Comment:
It's a bit confusing that the transition API refers to this concept as `previousOnTop` but the internal implementation uses the name `belowPrevious`. Couldn't they be the same?

Reply:
Made the naming consistent as `previousOnTop` everywhere.